src/ui.js

```diff
@@ -19,8 +19,26 @@ export const makeTable = async () => {
   const companiesToDisplay = [];
   companiesToDisplay.push(COMPANIES_TABLE_HEADERS);
 
+  function numberWithSpaces(x) {
+    return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, " "); //every three digits, add a space. it uses a lookahead regex.
+  }
+
+  //const hourRegex = //g
+
+  function dateTo24HourFormat(dateString){
+    let time = new Date(dateString)
```

Comment:
This is unfortunately not optimal solution. The reason is - if date format changes on backend, your frontend code will have to change, too.

The more robust way to implement this would be by using JS Date APIs. For example, check ‘toLocaleTime’ function - can it be applied here?

Reply:
Should I make a new branch now that I've made changes? Or just push and commit it to this same branch, and make a new PR?